.env.example

```
@@ -0,0 +1,4 @@
+OTP_API_BASE_URL=''
```

```suggestion
API_BASE_URL=''
```
No need really, it's obvious that this repo is about the OTP service